rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -366,13 +340,26 @@ This task tree matches the one shown earlier for merge-sort.
 
 ## Open Questions in Design
 
-Some open questions that remain:
-
+Questions that should be addressed before moving into `experimental` and providing a preview feature:
 - Are the suggested APIs sufficient?
+- Are there additional use cases that should be considered that we missed in our analysis?
+- Semantics for `task_arena::enqueue` method that takes a `task_handle` object should be defined for all possible states of the handle.
+- What are the performance targets for this feature?
+- What are the exit criteria for a feature to become fully supported?
+- The API applicability for other usage scenarios should be checked, e.g.:
+  * A wavefront example representing tasks with multiple successors and predecessors
+  * Fibonacci example
+  * A two-stage parallel scan
+  * Other non-trivial divide-and-conquer patterns
+
+Questions that should be addressed before the future movement into `supported` and having a fully supported feature:
 - Should we add a function to adds more than one predecessor as single call, such as `add_predecessors`?
 - Should we add functions that merge creation and definition of predecessor tasks, such as
-`template <typename Func> add_predecessor(Func&& f);`.
-- Are there additional use cases that should be considered that we missed in our analysis?
+`template <typename Func> add_predecessor(Func&& f);`?
+- Should we add a function that indicates that the task handled by `task_handle` object is completed, such as
+`bool task_handle::is_completed()`?
+- Should the API combining creating of the task and setting dependencies/transferring successors be added? Something like
+`task_handle::add_predecessor(Func)` where `Func` is the body of the predecessor.
```

Comment:
It sounds the same as the second item in the list.

Reply:
Thanks, removed